drivers/Makefile

```diff
@@ -1,12 +1,14 @@
 CC = arm-none-eabi-gcc
-CFLAGS = -c -Wall -I../include/ \
+CFLAGS = -c -Wall -std=gnu11 -Wformat=0 \
+	 -I../include/ \
 	 -I../projects/drivers/util/ \
 	 -I./dac/ad917x/ad917x_api/ \
 	 -I../projects/ad9361/src/ \
-	 -I./adc/ad9208/ad9208_api \
-	 -I../projects/drivers/axi_adc_core
+	 -I./adc/ad9208/ad9208_api/ \
+	 -I./axi_core/axi_adc_core/
 
-SUBDIRS = $(shell find . -path ./platform -prune -o -name '*.c')
+SUBDIRS = $(shell find . -path ./platform -prune -o \
+	  -path ./axi_core -prune -o -name '*.c')
```

Comment:
doing hard-coded skip stuff in Makefile looks messy; 
maybe add an env-var that make can use, which contains a list of folder/projects/stuff to skip